### PR TITLE
[search] Removed multiple viewports.

### PR DESCRIPTION
**Pull request by ygorshenin**
@mpimenov @bykoianko PTAL

### CODE REVIEW

File: search/processor.cpp [hunk 1]
```diff
@@ -210,9 +210,20 @@ void Processor::Init(bool viewportSearch)
...
+void Processor::SetViewport(m2::RectD const & viewport)
 {
-  SetViewportByIndex(viewport, CURRENT_V, forceUpdate);
+  ASSERT(viewport.IsValid(), ());
+
+  if (m_viewport.IsValid())
+  {
+    double constexpr epsMeters = 10.0;
+
+    // Skip if viewports are equal.
```
Comment (mpimenov) [inline search/processor.cpp]: This comment looks superfluous.

File: search/processor.hpp [hunk 2]
```diff
@@ -88,9 +86,11 @@ class Processor : public my::Cancellable
   inline m2::PointD const & GetPosition() const { return m_position; }
 
   /// Suggestions language code, not the same as we use in mwm data
-  int8_t m_inputLocaleCode, m_currentLocaleCode;
+  int8_t m_inputLocaleCode;
+  int8_t m_currentLocaleCode;
```
Comment (bykoianko) [inline search/processor.hpp]: Could you please init |m_inputLocaleCode| and |m_currentLocaleCode|?

Comment (bykoianko) [reply search/processor.hpp]: `#include <cstdint>`?

File: search/processor.cpp (hunk 1 above)
Comment (ygorshenin) [reply search/processor.cpp]: Done

File: search/processor.hpp (hunk 2 above)
Comment (ygorshenin) [reply search/processor.hpp]: Done

**Comment by ygorshenin** [comment]
@bykoianko PTAL

**Pull request merged**